db/migrate/20220504045733_create_alchemical_properties_canonical_ingredients.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class CreateAlchemicalPropertiesCanonicalIngredients < ActiveRecord::Migration[6.1]
```

Comment:
I thought about creating this as part of the previous migration since it's so closely related to that model. However, doing it more atomically allows us to roll back smaller changes at a time, so I went with multiple migrations instead.